aspnetcore/performance/caching/middleware.md

```diff
@@ -101,37 +94,39 @@ Response caching by the middleware is configured via HTTP headers. The relevant
 
 ## Caching respects request Cache-Control directives
 
-The middleware respects the rules of the [HTTP 1.1 Caching specification](https://tools.ietf.org/html/rfc7234#section-5.2). The rules require a cache to honor a valid `Cache-Control` header sent by the client. Under the specification, a client can make requests with a `no-cache` header value and force a server to generate a new response for every request. Currently, there's no developer control over this caching behavior when using the middleware because the middleware adheres to the official caching specification.
+The middleware respects the rules of the [HTTP 1.1 Caching specification](https://tools.ietf.org/html/rfc7234#section-5.2). The rules require a cache to honor a valid `Cache-Control` header sent by the client. Under the specification, a client can make requests with a `no-cache` header value and force the server to generate a new response for every request. Currently, there's no developer control over this caching behavior when using the middleware because the middleware adheres to the official caching specification.
 
-[Future enhancements to the middleware](https://github.com/aspnet/ResponseCaching/issues/96) will permit configuring the middleware for caching scenarios where the request `Cache-Control` header should be ignored when deciding to serve a cached response. If you seek more control over caching behavior, explore other caching features of ASP.NET Core. See the following topics:
+[Future enhancements to the middleware](https://github.com/aspnet/Home/issues/2612) will permit configuring the middleware for caching scenarios where the request `Cache-Control` header should be ignored when deciding to serve a cached response. For more control over caching behavior, explore other caching features of ASP.NET Core. See the following topics:
```

Comment:
Eliminate the word "will".

Reply:
Subject-verb agreement is a problem for dropping "will" [i.e., "future enhancements" and "permit" (present tense) don't align]. However, let's just drop this. It isn't going to be a 2.1 feature AFAICT. This amounts to fairly useless information at this point. That issue doesn't even clearly describe server-based caching features. I had one reader tell me, 'I don't understand how the linked issue applies.' They were asking about the feature offering server-controlled caching features.